DangProject/Sources/Presentation Layer/Setting/Alarm/View/AlarmDaySelectionView.swift

```diff
@@ -33,58 +32,79 @@ class AlarmDaySelectionView: UIView {
     var thursdayButton = UIButton()
     var fridayButton = UIButton()
     var saturdayButton = UIButton()
-    var tagNumber = 0
-    var delegate: AlarmDaySelectionDelegate?
-    
-    
+
     override init(frame: CGRect) {
         super.init(frame: frame)
-        if #available(iOS 15.0, *) {
             configureUI()
-        } else {
-            // Fallback on earlier versions
-        }
     }
     
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func configureButtonColor(_ tags: [Int]) {
+        resetAllButtons()
+        tags.forEach { tag in
+            switch tag {
+            case 1:
+                mondayButton.backgroundColor = .circleColorGreen
+                mondayButton.setTitleColor(UIColor.white, for: .normal)
+            case 2:
+                tuesdayButton.backgroundColor = .circleColorGreen
+                tuesdayButton.setTitleColor(UIColor.white, for: .normal)
+            case 3:
+                wednesdayButton.backgroundColor = .circleColorGreen
+                wednesdayButton.setTitleColor(UIColor.white, for: .normal)
+            case 4:
+                thursdayButton.backgroundColor = .circleColorGreen
+                thursdayButton.setTitleColor(UIColor.white, for: .normal)
+            case 5:
+                fridayButton.backgroundColor = .circleColorGreen
+                fridayButton.setTitleColor(UIColor.white, for: .normal)
+            case 6:
+                saturdayButton.backgroundColor = .circleColorGreen
+                saturdayButton.setTitleColor(UIColor.white, for: .normal)
+            case 7:
+                sundayButton.backgroundColor = .circleColorGreen
+                sundayButton.setTitleColor(UIColor.white, for: .normal)
+            default: break
+            }
+        }
+    }
 }
 
 extension AlarmDaySelectionView {
-    @available(iOS 15.0, *)
+    
     private func configureUI() {
         setUpStackView()
     }
     
-    @available(iOS 15.0, *)
     private func setUpStackView() {
-        let buttonsArray = [
-            sundayButton, mondayButton, tuesdayButton,
-            wednesdayButton, thursdayButton, fridayButton, saturdayButton
-        ]
-        buttonsArray.forEach {
-            $0.setAttributedTitle(
+        let buttonsArray = [mondayButton, tuesdayButton,
+            wednesdayButton, thursdayButton, fridayButton, saturdayButton, sundayButton]
+        var tagNumber = 1
+        buttonsArray.forEach { button in
+            button.setAttributedTitle(
                 NSAttributedString(
-                    string: "\(week[tagNumber])",
+                    string: "\(week[tagNumber-1])",
```

Comment:
```suggestion
                    string: "\(week[tagNumber - 1])",
```

이런거 보통 저는 띄어쓰는데 붙여쓰는걸로 통일인걸까요??

Reply:
반영했습니다
050622da159cb4d5b0b00d9bcf6358646f5874a9